stdlib/messaging/kafka/src/main/ballerina/src/kafka/Module.md

```diff
@@ -1,83 +1,15 @@
-## Module overview
-
 This module is used to interact with Kafka Brokers via Kafka Consumer and Kafka Producer clients.
 This module supports kafka 1.x.x and 2.0.0 versions.
 
-## Samples
-### Simple Kafka Consumer
```

Comment:
Shall we change the module.md structure similar to the other messaging modules. the code snippets to the usages.
1. https://github.com/daneshk/ballerina/blob/nats-docs-updates/stdlib/messaging/nats/src/main/ballerina/src/nats/Module.md
2. https://github.com/daneshk/ballerina/blob/rabbitmq-docs-updates/stdlib/messaging/rabbitmq/src/main/ballerina/src/rabbitmq/Module.md

Reply:
Fixed in d4730360c31d2d785c78586af3fd7631ca0a3506